docs/api/world.md

```diff
@@ -1,175 +1,358 @@
 # World
+A World contains entities which have components. The World is queryable and can be used to get entities with a specific set of components and to perform different kinds of operations on them.
 
-A World contains entities which have components. The World is queryable and can be used to get entities with a specific set of components.
+## Functions
+### new
+Creates a new world (necessary step to utilize the methods of the class)

```

Comment:
I find this change redundant. It doesn't actually add anything to the description. Just let the example show it.